recipe/meta.yaml

```diff
@@ -36,17 +36,18 @@ test:
 
 about:
   home: https://www.freetype.org/
-  license: GPL-2.0-only and LicenseRef-FreeType
+  license: GPL2
+  license_family: GPL2
   license_file:
     - docs/GPLv2.TXT
-    - docs/FTL.TXT
```

Comment:
docs/FTL.TXT is a correct file, see https://gitlab.freedesktop.org/freetype/freetype/-/blob/master/docs/FTL.TXT